Detectors/Calibration/include/DetectorsCalibration/Utils.h

```diff
@@ -22,17 +22,41 @@
 #include "Headers/DataHeader.h"
 #include "CommonUtils/StringUtils.h"
 #include "CCDB/CCDBTimeStampUtils.h"
+#include "CCDB/CcdbObjectInfo.h"
+#include "CommonUtils/MemFileHelper.h"
+#include "CCDB/CcdbApi.h"
 
 namespace o2
 {
 namespace calibration
 {
 
+using TFType = uint64_t;
+
 struct Utils {
   static constexpr o2::header::DataOrigin gDataOriginCDBPayload{"CLP"}; // generic DataOrigin for calibrations payload
   static constexpr o2::header::DataOrigin gDataOriginCDBWrapper{"CLW"}; // generic DataOrigin for calibrations wrapper
+  template <typename T>
+  static void prepareCCDBobjectInfo(T& obj, o2::ccdb::CcdbObjectInfo& info, const std::string& path, TFType tf,
+                                    const std::map<std::string, std::string>& md);
 };
 
+template <typename T>
+void Utils::prepareCCDBobjectInfo(T& obj, o2::ccdb::CcdbObjectInfo& info, const std::string& path, TFType tf,
```

Comment:
The TF is not a time, it cannot be used to populate the CCDB. Since these are generic methods, why don't you simply add explicit arguments for start/end time
```
  static void prepareCCDBobjectInfo(T& obj, o2::ccdb::CcdbObjectInfo& info, const std::string& path, 
                                    const std::map<std::string, std::string>& md, std::uint64_t tStart, std::unit64_t tEnd);
```

Reply:
Yes, I can change that too.